docs/development.rst

```diff
@@ -203,6 +191,42 @@ Then execute the following and copy the markdown version for pasting to GitHub r
 
 After the release commit has been pushed and a release has been tagged, set a development version in the same above files. This is basically the next minor release postfixed by ``-dev``.
 
+Developing with Docker
+------------------------
+
+If you choose, you may develop Socialhome using Docker, rather than installing Postfix and Redis manually on your computer.
+
+
+Supported versions
+..................
+
+This guide assumes you are running Docker on a GNU/Linux based system such as Ubuntu, Debian or Fedora Linux. It may be possible to run this on other platforms where Docker is supported, but those are untested.
+
+The docker development installation was tested on Docker version 17.09 and docker-compose 1.16.1.
+
+Steps
+......
+
+The first step is to copy the ``docker/dev/docker-compose.yml`` file to the root of the project. eg
+
+``cp docker/dev/docker-compose.yml .``
+
+You also need to set an .env file as per the above instructions. Use the ``.env.example`` as a starting point.
+
+From there, you can build the images:
+
+``docker-compose build``
+
+And then the steps you would normally do, but throught he django image, ala:
+
+``docker-compose run django manage migrate``
+and
+``docker-compose run django manage createsuperuser``
+
+And then just
+
+``docker-compose up``
+
```

Comment:
I wonder if it would make sense to also mention where the UI is exposed to, ie what to type in the browser to reach the running instance? Might help not so experienced developers. It's not apparent to me at least where I should look.

Reply:
Done. :heavy_check_mark: